doc/man3/flux_reactor_create.adoc

```diff
@@ -80,11 +80,9 @@ RETURN VALUE
 `flux_reactor_create()` returns a flux_reactor_t object on success.
 On error, NULL is returned, and errno is set appropriately.
 
-`flux_reactor_run()` returns zero on normal termination.
-On failure, termination by `flux_reactor_stop_error()` -1 is returned
-with errno set.  Additionally if either FLUX_REACTOR_NOWAIT or
-FLUX_REACTOR_ONCE is set and there are still watchers registered -1 is
-returned and errno set to EWOULDBLOCK.
+`flux_reactor_run()` returns the number of active watchers on
+successful termination.  On failure termination by
```

Comment:
How about
>  `flux reactor run()` runs the reactor loop until exit criteria are met.  Upon exit, the function returns the number of active watchers, or -1 if an error occurred.

Also I was just thinking on my ride home that maybe I should break out the `flux_reactor_run()` return value change to another PR that would preceed this one.  What do you think?